src/main/resources/init_db.sql

```diff
@@ -1,14 +1,119 @@
+  `user_id` BIGINT(11) NOT NULL,
+  `role_id` BIGINT(11) NOT NULL,
+  `deleted` TINYINT(1) NOT NULL DEFAULT 0,
+  INDEX `users_roles_users.fk_idx` (`user_id` ASC) VISIBLE,
+  INDEX `users_roles_roles.fk_idx` (`role_id` ASC) VISIBLE,
+  CONSTRAINT `users_roles_users.fk`
+    FOREIGN KEY (`user_id`)
+    REFERENCES `internet_shop`.`users` (`user_id`)
+    ON DELETE NO ACTION
+    ON UPDATE NO ACTION,
+  CONSTRAINT `users_roles_roles.fk`
+    FOREIGN KEY (`role_id`)
+    REFERENCES `internet_shop`.`roles` (`role_id`)
+    ON DELETE NO ACTION
+    ON UPDATE NO ACTION)
+ENGINE = InnoDB
+DEFAULT CHARACTER SET = utf8;
+
+CREATE TABLE `internet_shop`.`orders` (
+  `order_id` BIGINT(11) NOT NULL AUTO_INCREMENT,
+  `user_id` BIGINT(11) NOT NULL,
+  `deleted` TINYINT(1) NOT NULL DEFAULT 0,
+  PRIMARY KEY (`order_id`),
+  INDEX `order_user_id_users.fk_idx` (`user_id` ASC) VISIBLE,
+  CONSTRAINT `order_user_id_users.fk`
+    FOREIGN KEY (`user_id`)
+    REFERENCES `internet_shop`.`users` (`user_id`)
+    ON DELETE NO ACTION
+    ON UPDATE NO ACTION)
+ENGINE = InnoDB
+DEFAULT CHARACTER SET = utf8;
+
+CREATE TABLE `internet_shop`.`orders_products` (
+  `order_id` BIGINT(11) NOT NULL,
+  `product_id` BIGINT(11) NOT NULL,
+  INDEX `orders_products_order_id.fk_idx` (`order_id` ASC) VISIBLE,
+  INDEX `orders_products_product_id.fk_idx` (`product_id` ASC) VISIBLE,
+  CONSTRAINT `orders_products_order_id.fk`
+    FOREIGN KEY (`order_id`)
+    REFERENCES `internet_shop`.`orders` (`order_id`)
+    ON DELETE NO ACTION
+    ON UPDATE NO ACTION,
+  CONSTRAINT `orders_products_product_id.fk`
+    FOREIGN KEY (`product_id`)
+    REFERENCES `internet_shop`.`products` (`product_id`)
+    ON DELETE NO ACTION
+    ON UPDATE NO ACTION)
+ENGINE = InnoDB
+DEFAULT CHARACTER SET = utf8;
+
+CREATE TABLE `internet_shop`.`shopping_carts` (
+  `cart_id` BIGINT(11) NOT NULL AUTO_INCREMENT,
+  `user_id` BIGINT(11) NOT NULL,
+  `deleted` TINYINT(1) NOT NULL DEFAULT 0,
+  PRIMARY KEY (`cart_id`),
+  INDEX `cart_user_id_users.fk_idx` (`user_id` ASC) VISIBLE,
+  CONSTRAINT `cart_user_id_users.fk`
+    FOREIGN KEY (`user_id`)
+    REFERENCES `internet_shop`.`users` (`user_id`)
+    ON DELETE NO ACTION
+    ON UPDATE NO ACTION)
+ENGINE = InnoDB
+DEFAULT CHARACTER SET = utf8;
+
+CREATE TABLE `internet_shop`.`shopping_carts_products` (
+  `cart_id` BIGINT(11) NOT NULL,
+  `product_id` BIGINT(11) NOT NULL,
+  INDEX `carts_products_shopping_carts.fk_idx` (`cart_id` ASC) VISIBLE,
+  INDEX `carts_products_products.fk_idx` (`product_id` ASC) VISIBLE,
+  CONSTRAINT `carts_products_shopping_carts.fk`
+    FOREIGN KEY (`cart_id`)
+    REFERENCES `internet_shop`.`shopping_carts` (`cart_id`)
+    ON DELETE NO ACTION
+    ON UPDATE NO ACTION,
+  CONSTRAINT `carts_products_products.fk`
+    FOREIGN KEY (`product_id`)
+    REFERENCES `internet_shop`.`products` (`product_id`)
+    ON DELETE NO ACTION
+    ON UPDATE NO ACTION)
+ENGINE = InnoDB
+DEFAULT CHARACTER SET = utf8;
```

Comment:
we should insert ADMIN and USER role into our db, but I can't see corresponding query in init_db.sql